src/styles/main.scss

```diff
@@ -0,0 +1,135 @@
+@media (min-width: 300px) {
+  .container {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin-top: 50px;
+  padding: 50px, 40px;
```

Comment:
There is a mistake in the padding property. It should use spaces instead of commas to separate values, like `padding: 50px 40px;`.